### PR TITLE
Fix/3018 adblocker fallback

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue #3018

## Relevant technical choices

* Updates fallback mechanism to no longer follow redirects which is the default behavior for `fetch`
* Adds `timestamp` for cache busting to ensure we always make the request rather than serve from browser cache which would bypass detection
* Unfortunately `fetchMock` doesn't seem to handle redirects the same as the browser does so we can't add a test to cover this case specifically

**Before**
image

**After**
image

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/adsense/datastore/adblocker.js [hunk 1]
```diff
@@ -63,8 +68,21 @@ export const controls = {
...
+				addQueryArgs(
+					'/favicon.ico',
+					{
+						// The name of the parameter here doesn't really matter
+						// since adblockers look at the URL as a whole.
+						'google-site-kit': '/adsense/pagead2.googlesyndication.com/pagead/js/adsbygoogle.js',
+						// Add a timestamp for cache-busting.
+						timestamp: Date.now(),
+					}
+				),
```
Comment (eugene-manuilov) [inline assets/js/modules/adsense/datastore/adblocker.js:80]: The only thing that concerns me is that `addQueryArgs` escapes arguments which means that `/adsense/pagead2.googlesyndication.com/pagead/js/adsbygoogle.js` will be escaped to `%2Fadsense%2Fpagead2.googlesyndication.com%2Fpagead%2Fjs%2Fadsbygoogle.js`. However, I don't think it should bother adblocker and it should still detect it. :thinking:

**Review by eugene-manuilov — APPROVED** [review]
Going to approve it with one comment. Assigning to you, @aaemnnosttv, to take a look at my comment and merge this PR if you have tested it and it works for you.

File: assets/js/modules/adsense/datastore/adblocker.js (hunk 1 above)
Comment (aaemnnosttv) [reply assets/js/modules/adsense/datastore/adblocker.js]: Good catch! I found that certain adblockers such as uBlock Origin did not work as before when the value was encoded. I've removed the use of `addQueryArgs` which was mostly added to make the construction of the URL a bit easier to read. It now uses a simpler method and explicitly mentions the importance of not being encoded.

**Comment by aaemnnosttv** [comment]
Thanks for catching that @eugene-manuilov – back to you for another pass 👍

**Review by eugene-manuilov — APPROVED** [review]
Awesome! Thanks, @aaemnnosttv!

**Pull request merged**